src/main/java/com/artipie/YamlSettings.java

```diff
@@ -63,9 +70,51 @@ public Storage storage() throws IOException {
                 .yamlMapping("storage")
             );
         final String type =  yaml.string("type");
+        final Storage storage;
         if ("fs".equals(type)) {
-            return new FileStorage(Path.of(yaml.string("path")), Vertx.vertx().fileSystem());
+            storage = new FileStorage(Path.of(yaml.string("path")), Vertx.vertx().fileSystem());
+        } else if (type.equals("s3")) {
```

Comment:
@olegmoz let's make it `"s3".equals(type)`, or reverse previous if statement to `type.equals("fs")` just to be consistent.